testsuite/cluster-test/src/main.rs

```diff
@@ -251,17 +285,21 @@ impl ClusterUtil {
     }
 
     fn print_stat(&self, window: Duration) -> failure::Result<(f64, f64)> {
+        // WIP: What should be the behaviour here?
+        if self.prometheus.is_none() {
```

Comment:
Question: What should be the behaviour here when there is no Prometheus instance?

Reply:
We can just return error with `bail!`

Reply:
Consider `let my_prometheus = self.prometheus?;` or `let my_prometheus = self.prometheus.map_err(format_err!("<what you would pass as bail arg>"));`

Reply:
Cleaned up so that `self.prometheus` is not an Option<T> anymore.